src/sources/shopify/transform.js

```diff
@@ -1,19 +1,56 @@
-const { process: processV1 } = require('../../v1/sources/shopify/transform');
+const { process: processTrackerEvents } = require('./tracker/transform');
+const { getBodyFromV2SpecPayload } = require('../../v0/util');
+const { processPixelEvents } = require('./transformPixel');
+const { isShopifyV1Event } = require('./utils');
```

Comment:
```suggestion
const { isShopifyPixelEvent } = require('./utils');
```

Reply:
we already have v1 context in terms of API spec - again v1 in same file creates confusion